README.md

```diff
@@ -1,13 +1,30 @@
+<div align="center">
+<img src="https://cdn.discordapp.com/attachments/484717445538643979/564476620401016862/Banner.png">
```

Comment:
Maybe this image should be added to the repo ?

Reply:
It was already merged